ports/osgearth/portfile.cmake

```
@@ -55,6 +56,8 @@ vcpkg_cmake_configure(
         -DOSGEARTH_BUILD_TOOLS=OFF
```

```suggestion
        -DOSGEARTH_BUILD_TOOLS=OFF
        -DOSGEARTH_BUILD_EXAMPLES=OFF
```
and we save build time. And the port may no longer need to remove debug executables.